overlord/overlord.go

```diff
@@ -139,6 +134,10 @@ func New() (*Overlord, error) {
 
 	o.addManager(cmdstate.Manager(s))
 
+	if err := configstate.Init(hookMgr); err != nil {
```

Comment:
if I understand the intention, this should be configstateInit ?

Reply:
Heh, indeed, silly me. Fixed.